wolfcrypt/src/integer.c

```diff
@@ -4715,6 +4715,11 @@ int mp_prime_is_prime (mp_int * a, int t, int *result)
     return MP_VAL;
   }
 
+  if (mp_isone(a)) {
+      *result = 0;
```

Comment:
Why not MP_NO here?

Reply:
I fixed that one, and another in the same function where we used 1 instead of MP_YES.